host/sgx/loadelf.c

```diff
@@ -788,6 +870,19 @@ oe_result_t oe_load_elf_enclave_image(
         0)
         OE_RAISE(OE_FAILURE);
 
+    /* Save pointer to blob data and blob size */
+    {
+        elf64_t* elf = &image->u.elf.elf;
+        unsigned char* data;
+        size_t size;
+
+        if (elf64_find_section(elf, ".oeblob", &data, &size) == 0)
```

Comment:
should there be tooling in OE to add the oeblob section? Seems like an odd developer experience to tell people to add a section to their binary as they wish.